db/randomDataFuncs.js

```diff
@@ -0,0 +1,41 @@
+const fakeData = require('./fakeData.js');
+
+const pickRandomData = (targetData) => {
+  const range = targetData.length;
+  const randomNum = Math.floor(Math.random() * range);
+  return targetData[randomNum];
+};
+
+const oneToHundred = i => i % 100;
```

Comment:
would randomRestaurantId be more appropriate?

```suggestion
const randomRestaurantId = i => i % 100;
```

Reply:
If you commit this change remember to adjust line 25 as well.